### PR TITLE
Really use Cassandra Storage in HTTP Acceptance tests

### PR DESCRIPTION
**Pull request by Miles-Garnsey**
My previous PR seems to have lost the commits that switched to using Cassandra storage. This one re-instates this functionality.

It also switches to using a build of management API from the integration branch, now that the fixes to status changes have been merged.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -264,17 +264,13 @@ jobs:
...
         test-type: [ "http-api" ]
         grim-max: [ 1 ]
         grim-min: [ 1 ]
         # all versions but trunk have the same cucumber options, but we can't declare that more effectively (yet)
         include:
-          - cassandra-version: 'binary:3.11.11'
-            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags @http_management'
-            experimental: false
-            jdk-version: '8'
           - cassandra-version: 'binary:4.0.1'
```
Comment (adejanovski) [inline .github/workflows/ci.yaml:274]: **Question:** Is there something preventing us from testing against 3.11?

File: src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright 2014-2017 Spotify AB
- * Copyright 2016-2019 The Last Pickle Ltd
+ * Copyright 2017-2017 Spotify AB
```
Comment (adejanovski) [inline src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java:2]: **Question**: Do you know why that was changed?

File: src/server/src/test/resources/cassandra-reaper-http-at.yaml [hunk 3]
```diff
@@ -21,11 +21,11 @@ repairIntensity: 0.95
 scheduleDaysBetween: 7
 repairRunThreadCount: 15
 hangingRepairTimeoutMins: 1
-storageType: memory
+storageType: cassandra
 incrementalRepair: false
 blacklistTwcsTables: true
-enableDynamicSeedList: false
 jmxConnectionTimeoutInSeconds: 10
+activateQueryLogger: true
```
Comment (adejanovski) [inline src/server/src/test/resources/cassandra-reaper-http-at.yaml]: **nit:** Not sure we need this.

**Review by adejanovski — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Miles-Garnsey) [reply .github/workflows/ci.yaml:274]: Yes, aren't metrics unavailable on 3.11? Didn't we decide that HTTP management interface was only available on 4x?

File: src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java (hunk 2 above)
Comment (Miles-Garnsey) [reply src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java:2]: No, I have no idea on that sorry. It wasn't changed manually... Do you want it changed back?

File: src/server/src/test/resources/cassandra-reaper-http-at.yaml (hunk 3 above)
Comment (Miles-Garnsey) [reply src/server/src/test/resources/cassandra-reaper-http-at.yaml]: Removed.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (adejanovski) [reply .github/workflows/ci.yaml:274]: metrics are available post 3.11.13 IIRC. 
Let's worry about this once we've stabilized 4.x support.

File: src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java (hunk 2 above)
Comment (adejanovski) [reply src/server/src/test/java/io/cassandrareaper/acceptance/ReaperHttpIT.java:2]: I don't care that much, we should probably stop putting years (and even company names) anyway.

**Pull request merged**